aspen/http/request.py

```diff
@@ -226,6 +226,13 @@ def __new__(cls, method=b'GET', uri=b'/', server_software=b'',
 
             raise Response(400, "Request is undecodable. "
                                 "(%s:%d)" % (filename, frame.f_lineno))
+
+        # set up aliases
```

Comment:
Okay, but does that mean if someone changes `obj.path` that `obj.line.uri.path` will be changed?  Multiple sources of truth always worry me.  What about something like the `AliasAttribute` functionality described [here](http://code.activestate.com/recipes/276643-caching-and-aliasing-with-descriptors/?in=user-1732285) ?